dev-libs/msgpack/msgpack-1.2.0.ebuild

```diff
@@ -31,6 +31,12 @@ DOCS=( README.md )
 
 src_prepare() {
 	sed -i -e 's,-Werror,,g' CMakeLists.txt || die
+	sed -i -e 's,-Werror,,g' CMakeLists.txt || die
```

Comment:
an extra one? also maybe you should've wildcarded the dirs

Reply:
whoops, dumb copy and paste error.

will `**/*/CMakeLists.txt` work or do we need some kind of `find` magic?

Reply:
it should work